node/src/p2p/callbacks/p2p_callbacks_reducer.rs

```diff
@@ -290,6 +295,38 @@ impl crate::State {
                     best_tip: best_tip.clone(),
                 });
             }
+            P2pCallbacksAction::P2pPubsubValidateMessage {
```

Comment:
@0xMimir I would expect the block to be rejected here in the case of any error, but that is not happening, how does it work?